error.c

```diff
@@ -1467,12 +1467,22 @@ exc_init(VALUE exc, VALUE mesg)
 }
 
 /*
- * call-seq:
- *    Exception.new(msg = nil)        ->  exception
- *    Exception.exception(msg = nil)  ->  exception
+ *  call-seq:
+ *    Exception.new(message = self.to_s) ->  exception
```

Comment:
In the `Exception.new` context, `self.to_s` would mean the `to_s` of the class.

Reply:
Fixed?